### PR TITLE
added services section

### PR DESCRIPTION
**Pull request by hleete00**
added service section

### CODE REVIEW

File: components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+export const ServiceSectionCardHeading = styled.h1`
+  height: 30px;
+  margin-left: 20px;
+
+  font-family: "ITC Avant Garde Gothic Std";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 30px;
+  /* identical to box height */
```
Comment (JaredHightower) [inline components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.styles.tsx]: Let remove this comment

File: components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  width: 200px;
+  height: 78px;
+  margin-left: 20px;
+
+  font-family: "ITC Avant Garde Gothic Std";
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 26px;
+  /* or 162% */
```
Comment (JaredHightower) [inline components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.styles.tsx]: same here

File: components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  width: 200px;
+  height: 26px;
+  margin-left: 20px;
+
+  font-family: "ITC Avant Garde Gothic Std";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 26px;
+  /* identical to box height, or 173% */
```
Comment (JaredHightower) [inline components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.styles.tsx]: and here

File: components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import * as Styles from "./ServiceSectionCard.styles";
+import { SERVICE_SECTION_CARD_DATA } from "./ServiceSectionCard.const";
+
+export const ServiceSectionCard = () => {
+  console.log(SERVICE_SECTION_CARD_DATA);
```
Comment (JaredHightower) [inline components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.tsx]: remove console log here

File: components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+              {card.learn}
+            </Styles.ServiceSectionCardLearnMore>
+          </Styles.ServiceSectionCard>
+        );
+      })}
+    </>
+  );
+};
+
+//   <Styles.ServiceSectionCard key={SERVICE_SECTION_CARD_DATA[0].id}>
```
Comment (JaredHightower) [inline components/ServiceSection/ServiceSectionContent/ServiceSectionCard/ServiceSectionCard.tsx]: if this is unused lets remove

File: components/ServiceSection/ServiceSectionContent/ServiceSectionContent.styles.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+`;
+
+export const ServiceSectionDescription = styled.p`
+  width: 627px;
+  height: 56px;
+  margin: auto;
+  margin-top: 30px;
+  margin-bottom: 50px;
+
+  font-family: "ITC Avant Garde Gothic Std";
```
Comment (JaredHightower) [inline components/ServiceSection/ServiceSectionContent/ServiceSectionContent.styles.tsx]: Use the following font family for this `font-family:"ITC Avant Garde Gothic W01 Bk";` I have a paid font theme which is not correctly reflected in Figma.

**Review by JaredHightower — COMMENTED** [review]
Fantastic job on this man! I left a few comments for you to review, feel free to review them and let me know if you have any questions. Also, be sure to remove your yarn lock file. We should not need a yarn.lock and a package-lock.json file in the same project.

File: components/ServiceSection/ServiceSectionContent/ServiceSectionContent.styles.tsx (hunk 6 above)
Comment (hleete00) [reply components/ServiceSection/ServiceSectionContent/ServiceSectionContent.styles.tsx]: Do you want this just for the description below the heading or on the card descriptions as well?

Comment (JaredHightower) [reply components/ServiceSection/ServiceSectionContent/ServiceSectionContent.styles.tsx]: Yes

**Review by JaredHightower — APPROVED** [review]

**Pull request merged**